roboflow/models/classification.py

```diff
@@ -60,13 +72,19 @@ def predict(self, image_path, hosted=False):
             # Create API URL for hosted image (slightly different)
             self.api_url += "&image=" + urllib.parse.quote_plus(image_path)
             # POST to the API
-            resp = requests.get(self.api_url)
+            resp = requests.post(self.api_url)
+
+        image_dims = {"width": "0", "height": "0"}
```

Comment:
why we are sending back this information (with `zeros`)

Reply:
this was probably to fix a bug - and dims usually not relevant with classification

I'll dig in

Reply:
Copy, thanks. Resolving

Reply:
Unresolving because it tripped me up in the notebook that this was here & had wrong values (also they're strings not numbers?)